### ISSUE TITLE
Enable arrow keys to navigate through Secondary navigation menu

### ISSUE THREAD

**Issue by pranav300** [cerner/terra-dev-site#260]
# Feature Request

## Description

I think we should enable the arrow keys for navigating through the secondary navigation menu for better accessibility.

**Comment by dkasper-was-taken** [comment]
I believe the current implementation is not necessarily accessible with tabs, as link driven navigation generally not done with arrow keys, but it's using nested collapsed items similar to a tree view. Since the combination and secondary navigation in general isn't a standard accessible pattern we have a couple options.

First we could adjust the structure to have elements with the role of button using the aria-expanded attribute, instead of the 'aria-haspopup", leveraged for the drill in elements, and maintain the link role of the disclosure items. On selecting an end disclosure item focus should be set to the content area. Then we would need to include a "skip to content" type element at the top of the list.

Second we could switch it to a full tree view with list elements, which would include arrows and a single tab similar to this:
https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-1/treeview-1a.html
This would could forego the "skip to content" approach as a single tab would take you to the content.

The first option is probably quicker to implement and reaches a minimum level of accessibility, but the second option probably makes more sense in the long run, but will take considerably more work.
